crates/core/tedge_mapper/src/c8y/converter.rs

```diff
@@ -638,6 +638,7 @@ async fn execute_operation(
                 let logger = log_file.buffer();
                 let _result = child.wait_with_output(logger).await.unwrap();
             });
+
```

Comment:
I don't think we need this empty new line.

Reply:
removed